### ISSUE TITLE
to_if_alone doesn't work when Caps Lock is ON

### ISSUE THREAD

**Issue by danilagalimov** [pqrs-org/Karabiner-Elements#3461]
I have a configuration to remap left option to send the focus to Finder (i.e. like a Left Windows key). This perfectly works, but if I press Caps Lock, it stops working until Caps Lock Is turned off again. Please clarify what is wrong with my configuration:

```
{
                        "description": "Dock focus on Left Option",
                        "manipulators": [
                            {
                                "from": {
                                    "key_code": "left_option",
                                    "modifiers": {
                                        "mandatory": []
                                    }
                                },
                                "to": [
                                    {
                                        "key_code": "left_option",
                                        "lazy": true
                                    }
                                ],
                                "to_if_alone": [
                                    {
                                        "key_code": "f3",
                                        "modifiers": [
                                            "left_command",
                                            "left_shift",
                                            "left_control"
                                        ]
                                    }
                                ],
                                "type": "basic"
                            }
                        ]
                    }
```

**Comment by danilagalimov** [comment]
According to the event, this event (pressing left option) is simply not translated when caps lock is on.

**Comment by GeorgeeNico** [comment]
I have the same issue; my complex mods won't work when I have caps lock on. Anyway to fix? Am I missing a setting?

**Comment by michelkoga** [comment]
Try this:

'''
{
                        "description": "Dock focus on Left Option",
                        "manipulators": [
                            {
                                "from": {
                                    "key_code": "left_option",
                                    "modifiers": {
                                        "optional": [ caps_lock ]
                                    }
                                },
                                "to": [
                                    {
                                        "key_code": "left_option",
                                        "lazy": true
                                    }
                                ],
                                "to_if_alone": [
                                    {
                                        "key_code": "f3",
                                        "modifiers": [
                                            "left_command",
                                            "left_shift",
                                            "left_control"
                                        ]
                                    }
                                ],
                                "type": "basic"
                            }
                        ]
                    }
Ventura
'''

**Comment by danilagalimov** [comment]
Yes, now it works, thanks! But it seems that I have to include "optional": [ caps_lock ] into every configuration. I would like the option to treat Caps Lock as an ordinary key, not a modifier.

**Comment by danilagalimov** [comment]
Ideally it would be good to make this as a generic setting, i.e. treat caps lock as modifier
